home-assistant-plugin/custom_components/xknx/schema.py

```diff
@@ -113,11 +100,11 @@ class BinarySensorSchema:
                     cv.boolean,
                     cv.string,
                 ),
-                vol.Optional(CONF_IGNORE_INTERNAL_STATE, default=False): cv.boolean,
+                vol.Optional(CONF_IGNORE_INTERNAL_STATE, default=True): cv.boolean,
+                vol.Optional(CONF_CONTEXT_TIMEOUT, default=1): cv.string,
```

Comment:
```suggestion
                vol.Optional(CONF_CONTEXT_TIMEOUT, default=1): cv.positive_int,
```
or even better with float (0.5 seconds would be a viable option imho)
```suggestion
                vol.Optional(CONF_CONTEXT_TIMEOUT, default=1.0): vol.All(vol.Coerce(float), vol.Range(min=0, max=10)),
```